input/pagecontent/conformance.md

```diff
@@ -0,0 +1,163 @@
+*                                                                                       *
+* This file is generated by SUSHI. Any edits you make to this file will be overwritten. *
+*                                                                                       *
+* To change the contents of this file, edit the original source file at:                *
+* fhir-mCODE-ig\input\pagecontent\conformance.md                                        *
+*****************************************************************************************
+{% endcomment %} -->
+<!-- conformance.md {% comment %}
+*****************************************************************************************
+*                            WARNING: DO NOT EDIT THIS FILE                             *
+*                                                                                       *
+* This file is generated by SUSHI. Any edits you make to this file will be overwritten. *
+*                                                                                       *
+* To change the contents of this file, edit the original source file at:                *
+* fhir-mCODE-ig\input\pagecontent\conformance.md                                        *
+*****************************************************************************************
+{% endcomment %} -->
+<!-- conformance.md {% comment %}
+*****************************************************************************************
+*                            WARNING: DO NOT EDIT THIS FILE                             *
+*                                                                                       *
+* This file is generated by SUSHI. Any edits you make to this file will be overwritten. *
+*                                                                                       *
+* To change the contents of this file, edit the original source file at:                *
+* fhir-mCODE-ig\input\pagecontent\conformance.md                                        *
+*****************************************************************************************
+{% endcomment %} -->
+<!-- conformance.md {% comment %}
+*****************************************************************************************
+*                            WARNING: DO NOT EDIT THIS FILE                             *
+*                                                                                       *
+* This file is generated by SUSHI. Any edits you make to this file will be overwritten. *
+*                                                                                       *
+* To change the contents of this file, edit the original source file at:                *
+* fhir-mCODE-ig\input\pagecontent\conformance.md                                        *
+*****************************************************************************************
+{% endcomment %} -->
```

Comment:
It looks like this is getting added in multiple times by a script (SUSHI bug?). We should manually clean up until the bug is fixed.

Reply:
[This is a SUSHI bug](https://github.com/FHIR/sushi/issues/626), and is fixed by https://github.com/FHIR/sushi/pull/631 (confirmed).

I'm manually fixing this for now....we just need to be careful to continue to delete until the SUSHI binary is updated.